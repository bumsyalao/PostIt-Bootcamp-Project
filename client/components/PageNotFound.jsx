import React from 'react';
import NavigationBar from './NavigationBar';

const PageNotFound = () => (
  <div>
  <NavigationBar/>
    <h4 className="error-page"> Error 404: Page Not Found </h4>
  </div>

);

export default PageNotFound;
