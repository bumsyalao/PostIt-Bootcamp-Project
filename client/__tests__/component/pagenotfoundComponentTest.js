/* global expect jest test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import adapter from 'enzyme-adapter-react-16';
import PageNotFound from '../../components/PageNotFound';

configure({ adapter: new adapter() });

describe('PageNotFound component', () => {
  const spy = jest.fn();
  beforeEach(() => {
    global.Materialize = { toast: jest.fn() };
  });
  afterEach(() => {
    spy.mockReset();
  });

  const component = shallow(
    <PageNotFound />
  );

  test('should match the PageNotFound component snapshot', () => {
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
