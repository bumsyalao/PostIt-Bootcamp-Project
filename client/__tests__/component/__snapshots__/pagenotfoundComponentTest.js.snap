// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component should match the PageNotFound component snapshot 1`] = `
<div>
  <h4
    className="error-page"
  >
     Error 404: Page Not Found 
  </h4>
</div>
`;
